client_internals_test_utils.go

```diff
@@ -54,6 +54,10 @@ func (ci *ClientInternal) SerializationService() *serialization.Service {
 	return ci.client.ic.SerializationService
 }
 
+func (ci *ClientInternal) ProxyManagerProxies() map[string]interface{} {
+	return ci.client.proxyManager.proxies
```

Comment:
`proxyManager.proxies` is protected with a mutex. It should be used to access and copy the proxies before returning it to the user. i would add the `Proxies()` method to `proxyManager` and call it from here.

Reply:
Resolved, [0042502](https://github.com/hazelcast/hazelcast-go-client/pull/772/commits/0042502d165af4bbfd57b99474f816dd147c4599).